src/jvmTest/kotlin/bdd/SpaceEngineersCucumberTest.kt

```diff
@@ -197,9 +197,10 @@ class SpaceEngineersCucumberTest {
 
     @When("Character selects block {string} and places it.")
     fun character_places_selects_block_and_places_it(blockType: String) {
-        val toolbarLocation = ToolbarLocation(0, 0)
+        val toolbarLocation = ToolbarLocation(4, 0)
```

Comment:
In theory, yes. But there is not test relying on side effects of this call, is there? (And if yes, it should not be, I guess.)

Reply:
I think there is a convention about toolbar ordering, and usually tools are at the beginning... welder, grinder, drill (not sure about the ordering within), and then maybe riffle...

It probably does not matter much, but maybe we should follow the convention in our tests...

Reply:
It is more complicated. If you notice the code, it actually does not care, where is the block in toolbar, because it just puts it there at 0,4 or 0,0. Instead, it should just find out, where in toolbar it is and just select it and equip.

Also notice, that step "When Character sets toolbar slot 4, page 0 to "<blockType>"." Is happening after, so it is basically useless from this point of view.
Flow should be:

- Character sets toolbar slot/page to block type.
- Character selects block and places it without need of setting toolbar (either it is found in toolbar or there's error).

Anyway it's basically my fault + it is related to our old api not being able to get toolbar state. I will fix this in separate commit.

Reply:
Yeah, I actually noticed it does not care about previous content of the toolbar. That's why I dared to change it...